win32-mutex.gemspec

```diff
@@ -1,6 +1,7 @@
+
 Gem::Specification.new do |spec|
   spec.name       = "win32-mutex"
-  spec.version    = Win32::Mutex::VERSION
+  spec.version    = "0.4.4"
```

Comment:
This shouldn't be changed. We don't want to hardcode the version.

Reply:
Reverted the changes and modified version.rb to fix the issue.